### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by adamltyson**

### CODE REVIEW

**Review by vigji — APPROVED** [review]
Neat!

File: ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Please attach the log file if relevant (e.g. `output/cellfinder_2019-12-17_13-29-46.log`)
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+
+- OS: [e.g. Linux, Windows, MacOS]
+- Version [e.g. 18.04]
+- Hardware specs [e.g. 128GB RAM, Xeon 4114 CPU]
```
Comment (paddyroddy) [inline ISSUE_TEMPLATE/bug_report.md:27]: Probably good to make it clear on chip architecture, like we know `arm64` for `macOS` at least is a pain in the 🍑

File: ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Please enter full details of how to reproduce this bug.
+
+**Expected behavior**
```
Comment (paddyroddy) [inline ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
**Expected behaviour**
```

**Review by paddyroddy — APPROVED** [review]

**Comment by adamltyson** [comment]
@paddyroddy I'm getting errors running `pre-commit` locally. Any idea why I need `GLIBC_2.28` specifically?

```
[INFO] Installing environment for https://github.com/pre-commit/mirrors-prettier.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/adam/.cache/pre-commit/repoefj0gppp/node_env-default/bin/node', '/home/adam/.cache/pre-commit/repoefj0gppp/node_env-default/bin/npm', 'install', '--dev', '--prod', '--ignore-prepublish', '--no-progress', '--no-save')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    /home/adam/.cache/pre-commit/repoefj0gppp/node_env-default/bin/node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /home/adam/.cache/pre-commit/repoefj0gppp/node_env-default/bin/node)
```

**Review by FedeClaudi — APPROVED** [review]

**Pull request merged**